utils/runWebpack.js

```diff
@@ -20,8 +20,7 @@ const path = require('path');
 const files = [
   path.join('src', 'injected', 'zsSelectorEngine.webpack.config.js'),
   path.join('src', 'injected', 'selectorEvaluator.webpack.config.js'),
-  path.join('src', 'web.webpack.config.js'),
-];
+s];
```

Comment:
This is probably the reason all bots are red.

Reply:
thanks